torchgeo/datamodules/l8biome.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8Biome datamodule."""
+
+from typing import Any
+
+import torch
+
+from ..datasets import L8Biome
+from .geo import GeoDataModule
+
+
+class L8BiomeDataModule(GeoDataModule):
+    """LightningDataModule implementation for the L8 Biome dataset.
+    
+    .. versionadded:: 0.5
+    """
+
+    mean = torch.tensor(0)
+    std = torch.tensor(10000)
```

Comment:
This is fine for now, although we'll likely replace this with the SSL4EO-L8 mean and std dev once we finish downloading that dataset.